src/status_im2/common/home/banner/view.cljs

```diff
@@ -11,6 +12,8 @@
             [status-im2.common.home.view :as common.home]
             [utils.re-frame :as rf]))
 
+(def card-banner-overflow (reagent/atom :visible))
```

Comment:
while I'm sure there already many in the codebase, imo we should still avoid adding global ratoms like this.
Is it possible to make this local to the component using it?

cc: @ulisesmac, @ilmotta

Reply:
@J-Son89 I acknowledge the concern, but due to the atom's usage and changes happen in different area and, local usage isn't feasible. If you have any suggestions on how to address this, I'd greatly appreciate your input.

Reply:
If this is possible with passing props but it would be horrible to do so, then perhaps we can create a react context to make this ratom available in it's nested components/ where it's being used? 🤔

Reply:
I don't know, it's always feasible to use local ratoms. If the state is "too global" and we want to avoid prop drilling, we can just move the state to the re-frame layer.

Reply:
Yeah, using the scroll input here seems to have a worse impact, especially given that this atom is only used in the banner section. I've taken care to ensure that we only change its value when necessary to avoid any potential performance issues.

Reply:
as I discussed with @J-Son89 i'm going to close this PR with current approach until we find a better solution for this topic

Reply:
Sure @mohsen-ghafouri, thanks for the clarification.